add section troubleshoot related issue btw kubectl/kustomize on k8s deploy (#3573)
airbytehq/airbyte: docs/deploying-airbyte/on-kubernetes.md

* Google Kubernetes Engine \(GKE\)
* Amazon Elastic Kubernetes Service \(EKS\)

> Please let us know on [Slack](https://slack.airbyte.io) or with a Github Issue if you're having trouble running it on these or other platforms.
> We'll be glad to help you get it running.

Reviewer: ```suggestion
Please let us know on [Slack](https://slack.airbyte.io) or with a [Github Issue](https://github.com/airbytehq/airbyte/issues/new/choose) if you're having trouble running it on these or other platforms - we'll be glad to help you get it running.
```

   * If you redeploy `airbyte-webapp`, you will need to re-run this process.
6. Go to [http://localhost:8000/](http://localhost:8000/) and use Airbyte!

> If you faced issues on launching Airbyte on Kubernetes, you can check the troubleshooting section below.

Reviewer: ```suggestion
If you have faced issues on launching Airbyte on Kubernetes, check the troubleshooting section below.
```

kubectl exec -it airbyte-scheduler-6b5747df5c-bj4fx cat /tmp/workspace/8/0/logs.log
```

## Troubleshooting

**Getting error: json: unknown field "envs" after running `kubectl apply -k kube/overlays/stable`**

> This is a problem between `kubectl` and `kustomize` mismatch versions and can happen when
> you are using a cloud provider who didn't update `kubectl` yet.

Reviewer: ```suggestion
This is a version mismatch between `kubectl` and `kustomize` which can happen when you are using a cloud provider running on older versions of `kubectl`. To fix this, update your `kubectl` version.
```

> You should update your kubectl version.

Reviewer: ```suggestion
```

The current deployment in this guide was tested using kubectl 1.21.+.

> You can read about the issue [here](https://github.com/kubernetes-sigs/kustomize/issues/1069).

Reviewer: ```suggestion
You can read more about the issue [here](https://github.com/kubernetes-sigs/kustomize/issues/1069).
```
